way_to_home/custom_user/models.py

```diff
@@ -76,3 +76,13 @@ def create(cls, email, password, google_token='', phone_number=''):
             return user
         except (ValueError, IntegrityError):
             pass
+
+    @classmethod
+    def delete_by_id(cls, obj_id):
+        """Delete object, found by id."""
+        try:
+            objct = cls.objects.get(id=obj_id)
+            objct.delete()
```

Comment:
Rename it to "object", or maybe to "user"

Reply:
yes, of course